kim/Tiltfile

```diff
@@ -0,0 +1,29 @@
+
+# -*- mode: Python -*-
+
+def kim_build(ref, context, ignore=None, extra_flags=None):
+  """Use kim (https://github.com/rancher/kim) to build images for Tilt.
+  Args:
+    ref: The name of the image to build. Must match the image
+      name in the Kubernetes resources you're deploying.
+    context: The build context of the binary to build. Expressed as a file path.
+    ignore: Changes to the given files or directories do not trigger rebuilds.
+      Does not affect the build context.
+    extra_flags: Extra flags to pass to kim build. Expressed as an argv-style array.
+  """
+  extra_flags = extra_flags or []
+  extra_flags_str = ' '.join([shlex.quote(f) for f in extra_flags])
+
+  push_cmd = "echo already pushed $EXPECTED_REF"
+
+  custom_build(
+    ref=ref,
+    command=(
+      "set -ex\n" +
+      "kim build -t $EXPECTED_REF %s %s\n" +
```

Comment:
I think the "already pushed" message is redundant (see below), but can you also remove the `set -ex`? Tilt will fail the build as-is if `kim` returns a non-zero exit code and it shows the command it's going to run, so this is redundant.

But more importantly, making this _just_ `kim build ...` will make sure it works on Windows as well without a separate `command_bat`.